configs/model_config.py

```diff
@@ -8,26 +8,42 @@
 
 def mobilevit_xxs_config() -> ml_collections.ConfigDict:
     configs = ml_collections.ConfigDict()
-    configs.expansion_factor = 2  
-    configs.num_blocks = [2, 4, 3]  
+    configs.expansion_factor = 2
+    configs.num_blocks = [2, 4, 3]
     configs.projection_dims = [64, 80, 96]
-    configs.out_channels = [16, 16, 24, 24, 48, 48, 64, 64, 80, 80, 320]  
+    configs.out_channels = [16, 16, 24, 24, 48, 48, 64, 64, 80, 80, 320]
     return configs
```

Comment:
Mention the source of these configs.

Reply:
I have mentioned the configs where they have been taken from.

```
Configurations for different MobileViT variants. 
Referred from: https://arxiv.org/abs/2110.02178
```

Reply:
May be also mention the section in the paper?